pkg/project/project_test.go

```diff
@@ -36,6 +34,10 @@ func TestDelete(t *testing.T) {
 
```

Comment:
Could you tell me how these two cases are different? Don't see anything different.

Reply:
Need negative cases too.

Reply:
> Could you tell me how these two cases are different? Don't see anything different.

Deleting the only remaining project does not set any other project as active but deleting a project when there are other existing projects does. I think the sentence wasa not conveying the intended meaning fixed it now..

> Need negative cases too.

No mockable/testable negative cases here bcoz, based on https://github.com/redhat-developer/odo/pull/648/files#r216311973, the validations were moved to cmd/project.go that carries no UTs.

Reply:
Negative case: empty project list.

Reply:
Empty project is validated a layer above. So, that case although will not fail here(pkg/project/project.go), is treated as good as a normal case

Reply:
Unit tests LGTM, I guess the inexistent project validation part is kept under `cmd/project.go` is because of the delete confirmation part.